style.css

```diff
@@ -347,6 +348,8 @@ cite {
     margin-bottom: 0.5rem;
     font-weight: bold;
     color: black;
+    background-color: rgba(243, 248, 247, 0.287);
```

Comment:
Comment: Added a background colour under the label texts for better readability.